src/stp/plays/TestPlay.cpp

```diff
@@ -5,6 +5,7 @@
 #include "stp/plays/TestPlay.h"
 
 #include "stp/roles/TestRole.h"
+#include "stp/roles/Halt.h"
```

Comment:
You should probably revert all the changes in this file, seems they are just for testing.